Splitio.Redis/Services/Cache/Classes/RedisImpressionsCache.cs

```diff
@@ -29,7 +30,7 @@ public RedisImpressionsCache(IRedisAdapterProducer redisAdapter,
             string machineIP,
             string sdkVersion,
             string machineName,
-            string userPrefix = null) : base(machineIP, sdkVersion, machineName, userPrefix)
+            RedisConfig redisConfig = null, bool clusterMode = false) : base(machineIP, sdkVersion, machineName, redisConfig, clusterMode)
```

Comment:
you can remove machineName, machineIP and sdkVersion and you could use redisConfig